### ISSUE TITLE
Bug: Some scenes were found to be untranslatable into the corresponding Chinese.

### ISSUE THREAD

**Issue by lispking** [google/mdbook-i18n-helpers#235]
## **po file**: 
https://github.com/lispking/walrus-docs/blob/main/po/zh_CN.po#L251
![Image](https://github.com/user-attachments/assets/7420d90c-4891-4661-a620-d3e1f2c4881a)

## **Effect**:
![Image](https://github.com/user-attachments/assets/11577811-2239-4d9f-a56b-0b1ad11aee4a)

**Comment by mgeisler** [comment]
Hi @lispking, thanks for opening the issue!

Okay, this looks strange indeed and I'm not sure why this wouldn't work. I guess you can confirm that your `zh_CN.po` file has an up-to-date `msgid` for this text?

I think we could use a debug command which can help show us the text we find in the Markdown files. That would help us track down if we're looking up the right message in the `.po` file.

**Comment by lispking** [comment]
Hello @mgeisler , There is a corresponding msgid.

I don't know if it's related to the following line.

```admonish tip title=\"Fun fact\"\n"

**Comment by mgeisler** [comment]
Yes, it might be related to "info string" after the ` ``` `. We are supposed to pass those though unchanged, but we might have a bug there, I'm not sure 😄

I won't have time to debug this myself right now, but I hope I can find someone with some free time to look at this in the coming weeks.
